.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
a little wonky here but apparently prettier NEEDS a config file somewhere or else it won't format anything